.github/workflows/ci.yml

```
@@ -1,30 +1,127 @@
----
-
 name: CI
 
 on:
   pull_request:
-    branches: [devel]
-
   push:
-    branches: [devel]
+
 
 jobs:
+  sanity:
+    name: ${{ matrix.test.name }}
+    runs-on: ubuntu-20.04
+    container:
+      image: quay.io/ansible/ansible-builder-test-container:1.0.0
+      env:
+        PIP_CACHE_DIR: ${{ runner.temp }}/.cache/pip
+        TOXENV: ${{ matrix.test.tox_env }}
+
+    strategy:
+      fail-fast: false
+      matrix:
+        test:
+          - name: Lint
+            tox_env: linters
+
+          - name: Docs
+            tox_env: docs
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Create tox environment
+        run: tox --notest
```

A common theme you may notice across the popular project CI setups is that they use `--skip-missing-interpreters false` in every tox invocation, as a sanity check.
```suggestion
        run: tox --skip-missing-interpreters false --notest
```